tutorial-1-basic-nacl/westpa_scripts/run_gromacs.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
+import re
+import MDAnalysis as mda
+from MDAnalysis.analysis.distances import self_distance_array
+import numpy as np
+from imdclient.IMD import IMDReader
+import socket 
+import time
+
+proc =subprocess.Popen(["gmx_imd", "mdrun", "-s", "seg.tpr", 
```

Comment:
You should also check if the process already exited with a non-zero exit code.

Basically, making this a bit more robust.